backends/cuda-gen/ceed-cuda-gen-operator.c

```diff
@@ -112,6 +112,22 @@ static int CeedOperatorApplyAdd_Cuda_gen(CeedOperator op, CeedVector input_vec,
   CeedCallBackend(CeedOperatorGetFields(op, &num_input_fields, &op_input_fields, &num_output_fields, &op_output_fields));
   CeedCallBackend(CeedQFunctionGetFields(qf, NULL, &qf_input_fields, NULL, &qf_output_fields));
 
+  // Check for tensor-product bases
+  {
+    bool has_tensor_bases;
+
+    CeedCallBackend(CeedOperatorHasTensorBases(op, &has_tensor_bases));
+    // -- Fallback to shared if not all bases are tensor-product
```

Comment:
Should probably update the comment too, to avoid confusing future selves :)

Reply:
thanks